plugin/pkg/admission/priority/admission.go

```diff
@@ -44,6 +44,8 @@ const (
 )
 
 // SystemPriorityClasses defines special priority classes which are used by system critical pods that should not be preempted by workload pods.
+// NOTE: In order to avoid conflict of names with user-defined priority classes, all the names must
+// start with scheduling.SystemPriorityClassPrefix which is by default "system-".
 var SystemPriorityClasses = map[string]int32{
 	"system-cluster-critical": SystemCriticalPriority,
 	"system-node-critical":    SystemCriticalPriority + 1000,
```

Comment:
Should we manage the magic numbers like 1000? At least we should indicate why it is 1000. 
WDYT, Bobby?

Reply:
@wgliang, the exact value of priority does not have any significance, only the relative value of priority compared to other priorites matters. So, if `system-node-critical` value was `SystemCriticalPriority + 1`, it would have had the same effect as `SystemCriticalPriority + 1000`, but we choose to use a larger number so that we can add more priority classes between the two existing ones if there is a need in the future.